### ISSUE TITLE
[objc] SWGApiClient class calls deprecated AFNetworking function

### ISSUE THREAD

**Issue by edwd** [swagger-api/swagger-codegen#7644]
##### Description

The SWGApiClient class makes two calls to a deprecated AFNetworking function. The deprecated function is `dataTaskWithRequest:completionHandler:`. This results in an Xcode deprecation warning.

This function was deprecated in AFNetworking 3. In AFNetworking, you can see that this function was left in as a stub which calls its replacement function, `dataTaskWithRequest:uploadProgress:downloadProgress:completionHandler:`. (The stub passes in `nil` values for the two new parameters that were not present in the original function.) 

##### Swagger-codegen version

2.4

##### Command line used for generation

`java -jar swagger-codegen-cli.jar generate -i ~/petstore.yaml -l objc -o ~/PetStoreClient`

##### Steps to reproduce

 1. Generate an Objective-C client.
 2. Add client to Xcode project according to the instructions under __Install from local path using CocoaPods__ in the generated README.md.
 3. From the project directory, run `pod update`.
 4. Open and build the project in Xcode.
 5. The result is two deprecation warnings.

##### Related issues/PRs

N/A

##### Suggest a fix/enhancement

The calls to the deprecated function are generated at lines 93 and 115 of __ApiClient-body.mustache__:
 - https://github.com/swagger-api/swagger-codegen/blob/master/modules/swagger-codegen/src/main/resources/objc/ApiClient-body.mustache#L93
 - https://github.com/swagger-api/swagger-codegen/blob/master/modules/swagger-codegen/src/main/resources/objc/ApiClient-body.mustache#L115

These should be changed __FROM__:
```objc
    NSURLSessionDataTask* task = [self dataTaskWithRequest:request completionHandler:^(NSURLResponse *response, id responseObject, NSError *error) {
```
__TO__:
```objc
    NSURLSessionDataTask *task = [self dataTaskWithRequest:request uploadProgress:nil downloadProgress:nil completionHandler:^(NSURLResponse * _Nonnull response, id  _Nullable responseObject, NSError * _Nullable error) {
```

In other words, we should do what AFNetworking is already doing on line 716 of __AFURLSessionManager.m__, which will remove the deprecation warning during build:
 - https://github.com/AFNetworking/AFNetworking/blob/3.2.0/AFNetworking/AFURLSessionManager.m#L716
